src/partials/language/et.html

```diff
@@ -0,0 +1,41 @@
+<!--
+  Copyright (c) 2016-2019 Martin Donath <martin.donath@squidfunk.com>
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to
+  deal in the Software without restriction, including without limitation the
+  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+  sell copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+  IN THE SOFTWARE.
+-->
+
+<!-- Translations: Estonian -->
+{% macro t(key) %}{{ {
+  "language": "et",
+  "clipboard.copy": "Kopeeri lõikelauale",
+  "clipboard.copied": "Kopeeritud",
+  "edit.link.title": "Muuda seda lehte",
+  "footer.previous": "Eelmine",
+  "footer.next": "Järgmine",
+  "meta.comments": "Kommentaarid",
+  "meta.source": "Lähtekood",
+  "search.placeholder": "Otsi",
```

Comment:
Great PR! Just one little thing needs to be added to disable stemming (because by default, the English stemmer is used):

https://github.com/squidfunk/mkdocs-material/blob/0877168d111b02c1d70a3ecfd6b6ff505e7dd57d/src/partials/language/gr.html#L33-L35

Reply:
+ also in the built file (in the `material` folder), obviously

Reply:
Done :slightly_smiling_face: